README.md

```diff
@@ -40,6 +40,46 @@ kubectl delete -f config/samples/astraconnector_v1.yaml -n astra-connector
 ```
 kubectl delete -f astraconnector_operator.yaml -n astra-connector-operator
 ```
+
+## Security Best Practices
+### Use RBAC
+Use RBAC (Role Based Access Control) in Kubernetes to limit access to the connector namespace after deployment
+
+### Use a service mesh
+Use a service mesh to secure communications between pods
+
+### Limit token usage for registration
+The API token used for registration is only used once, and is not used after registering the connector.
+If the token was created specifically for registering the connector, we recommend discarding it after registration is complete.
+The token is not use by the connector after registration.
```

Comment:
I would say remove it from the CRD and revoke it from Astra. If its still in the CRD and the operator reconciles due to some change event, it will keep complaining about an invalid token. From the private cluster side, there is no way to tell if the locationID has been set in the cloud-extensions DB without making an API call.

Reply:
Good catch. I've updated the documentation.